shell/platform/common/client_wrapper/include/flutter/encodable_value.h

```diff
@@ -215,6 +215,11 @@ class EncodableValue : public internal::EncodableValueVariant {
     }
     return std::get<int64_t>(*this);
   }
+
+  // Avoid operator<=> problems with std::variant in C++20 mode.
+  friend bool operator<(const EncodableValue& lhs, const EncodableValue& rhs) {
+    return static_cast<const super&>(lhs) < static_cast<const super&>(rhs);
```

Comment:
I'm not following this code. What does an explicit cast to a `super` reference do? This needs an explanatory comment.